lib/block-supports/border.php

```diff
@@ -58,12 +58,21 @@ function gutenberg_apply_border_support( $block_type, $block_attributes ) {
 	) {
 		$border_radius = $block_attributes['style']['border']['radius'];
 
-		// This check handles original unitless implementation.
-		if ( is_numeric( $border_radius ) ) {
-			$border_radius .= 'px';
+		if ( is_array( $border_radius ) ) {
+			// We have individual border radius corner values.
+			foreach ( $border_radius as $key => $radius ) {
+				// Convert CamelCase corner name to kebab-case.
+				$corner   = strtolower( preg_replace( '/(?<!^)[A-Z]/', '-$0', $key ) );
+				$styles[] = sprintf( 'border-%s-radius: %s;', $corner, $radius );
+			}
+		} else {
```

Comment:
Can the border radius be `null`? Do we need to check for that like we do for spacing?

Reply:
I believe the original code here for the border support was added prior to `_wp_array_get` getting wider usage so the code approaches the check a little differently to the spacing support. 

Along with checking for the feature support, it also does an `isset` check which would return false for a null value. Would making the supports' code consistent in approach be best done in a separate PR? I'm happy to take care of that in a follow up.